opensrp-reveal/src/main/java/org/smartregister/reveal/model/TaskDetails.java

```diff
@@ -384,4 +384,20 @@ public void setGroupedTaskCodes(String groupedTaskCodeStatusString) {
         }
         this.groupedTaskCodes = stringBuilder.toString();
     }
+
+    /**
+     * Check if the grouped task details have a completed family registration task in it.
+     * @return true if there's a completed registration, false otherwise
+     */
+    public boolean hasRegisteredFamily() {
+        return getFamilyMemberNames() != null && !getFamilyMemberNames().equalsIgnoreCase("non_registered");
+    }
+
+    /**
+     * @return the difference of taskCount and completedTaskCount for grouped tasks and 1 otherwise
+     */
+    public int getPendingTasksCount() {
+        if (getTaskCount() == null) return 1;
+        return getTaskCount() - getCompleteTaskCount();
```

Comment:
The method returns "the difference of taskCount and completedTaskCount for grouped tasks and 1 otherwise"

For MDA and FI tasks are grouped and therefore this needs to be calculated, for other plans the tasks aren't grouped and task count is null, so we can safely assume there's only 1 task and return that number.